sdk/core/core-tracing/README.md

```diff
@@ -0,0 +1,23 @@
+# Azure Core HTTP client library for JS
```

Comment:
Still has the Core HTTP readme. Can probably delete this for now?

Reply:
Updated